clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -426,9 +426,9 @@ class MarkDeviceFunction : public RecursiveASTVisitor<MarkDeviceFunction> {
 
       if (auto *A = FD->getAttr<IntelReqdSubGroupSizeAttr>())
         Attrs.insert(A);
-      else if (auto *A = FD->getAttr<ReqdWorkGroupSizeAttr>())
+      if (auto *A = FD->getAttr<ReqdWorkGroupSizeAttr>())
         Attrs.insert(A);
```

Comment:
Suggest merging these `if` statements and adding early exit.

Reply:
I can merge these ifs, but why early exit?

Reply:
No reason. Never mind :)

Reply:
Also I think that these ifs aren't really mergeable.

Reply:
Probably you are right.